charts/fluent-operator/templates/fluent-operator-clusterRole.yaml

```diff
@@ -31,6 +31,7 @@ rules:
       - events
     verbs:
       - list
+      - watch
```

Comment:
As I can remember the roles in charts and manifests are generated.
@wenchajun could you point out what else to modify?
cc @wanjunlei